content/04.results.md

```diff
@@ -2,13 +2,15 @@
 
 Results section stub.
 
-### The Open Pediatric Brain Tumor Atlas
+### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas
 
-This section will introduce the dataset (e.g., the histologies represented and what data types are included; Figure {@fig:openpbta-overview}A-B) and the process for contributing analytical code and to the manuscript (Figure {@fig:openpbta-overview}C-D).
+We previously performed WGS, WXS, and RNA-Seq on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) (@doi:10.1093/neuonc/noz192), samples sourced from the Children’s Brain Tumor Network (CBTN) and the PNOC003 DMG clinical trial (@doi:10.1002/ijc.32258) of the Pacific Pediatric Neuro-oncology Symposium (PNOC) (*Figure 1*). 
+We then harnessed the benchmarking efforts of the KidsFirst Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses (variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions, *Figure S1* - red boxes and *STAR Methods*). 
+Next, we created a Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with CircleCI (Continuous Integration) to ensure analysis reproducibility and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot (@https://manubot.org/) integration to enable manuscript creation using Markdown within GitHub. 
+We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository (https://github.com/AlexsLemonade/OpenPBTA-analysis/#data-access-via-download-script) or open access CAVATICA project (https://cavatica.sbgenomics.com/u/cavatica/openpbta/).  
```

Comment:
I don't think these URLs need to be here necessarily. It strikes me as potentially confusing. I expect that the CAVATICA project will be in data availability?

Reply:
yeah you're right - they can go into data availability (worked on this first but should go there)